cosmwasm/packages/wasmi-runtime/src/cosmwasm/types.rs

```diff
@@ -102,48 +126,139 @@ pub struct Coin {
 }
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq)]
-#[serde(rename_all = "lowercase")]
-pub enum CosmosMsg {
+#[serde(untagged)]
+pub enum WasmOutput {
+    ErrObject {
+        #[serde(rename = "Err")]
+        err: Value,
```

Comment:
We don't really need to know exactly what type of error it is, since we just encrypt it all and wrap it in `generic_error`. So no need to use `StdError`.
As for the string errors, idk tbh, just refactored that code to be readable/maintainable.. you can open an issue if you want I guess